src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -789,8 +798,8 @@ void generateSummaryFiles(String timestamp) throws IOException {
     }
   }
 
-  Set<String> getPrecinctIds() throws IOException {
-    return precinctIds;
+  FieldIdSet getFieldIds() {
```

Comment:
Should `fieldIds`/`getFieldIds` clarify that it's referring to the _enabled_ ones? And why do we need the `fieldIds` instance variable instead of just using `config.enabledFields()`? Aren't we throwing an exception if there's a discrepancy between the config and the CVRs in this regard anyway?